ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationSaveOrAddStepConponent.java

```diff
@@ -0,0 +1,31 @@
+package io.syndesis.qe.pages.integrations.edit;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+/**
+ * Created by sveres on 11/29/17.
+ */
+public class IntegrationSaveOrAddStepConponent extends SyndesisPageObject {
```

Comment:
Typo in component. And I've a feeling that this class hierarchy is too granular.

Reply:
@dsimansk actually "syndesis-integrations-save-or-add-step" is only on the page "Add to Integration" page. Yet it is not child of "syndesis-integrations-connection-select" but only of "syndesis-integrations-edit-page" (main page). If I want to check presence via our standardized way (SyndesisPage), I think I don't have other option as to have separate component class for it.

Reply:
Hmm, ok then. But pls fix the typo in the class name :)

Reply:
@dsimansk done.